plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2933,3 +2933,42 @@ function wc_update_950_tracking_option_autoload() {
 	);
 	wp_set_option_autoload_values( $options );
 }
+
+/**
+ * Modifies PKs for woocommerce_order_itemmeta and wp_wc_orders_meta tables for more robust select operations.
+ */
+function wc_update_970_modify_primary_key_for_order_related_meta_tables() {
+	global $wpdb;
+
+	$prefix  = $wpdb->prefix;
+	$changes = array(
+		// NEXT: contradicts with the function name a bit.
+		'options'                    => array(
+			'original_pk' => 'option_id',
+			'query'       => "ALTER TABLE {$prefix}options ADD UNIQUE KEY option_id (option_id), DROP PRIMARY KEY, ADD PRIMARY KEY (option_name), DROP KEY option_name",
+		),
+		'woocommerce_order_itemmeta' => array(
```

Comment:
Benchmarking (5000 x 10 entries in the table):
Insert: 0.000289 -> 0.000300 seconds / query
Select: 0.000245 -> 0.000263 seconds / query

Not much value either.